src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,343 @@
+package duke;
+
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.io.FileNotFoundException;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.PrintWriter;
+
+public class Duke {
+
+    public static void main(String[] args) {
+        showWelcomeScreen();
+
+        ArrayList<Task> storeTasks = new ArrayList<>();
+        ArrayList<String> tasksText = new ArrayList<>();
+        String line;
+        Scanner in = new Scanner(System.in);
+        String filePath = "./data/duke.txt";
+
+        try {
+            readFileContents(filePath, storeTasks ,tasksText);
+        } catch (FileNotFoundException e) {
+            System.out.println("File not found");
+        } catch (DukeException e) {
+            System.out.println(e.getMessage());
+        }
+
+        while (true) {
+            listCommand(storeTasks);
+            line = in.nextLine();
+            if (line.equals("bye")) {
+                break;
+            }
+            if (line.equals("list")) {
+                listCommand(storeTasks);
+                continue;
+            }
+            if (line.contains("done")) {
+                try {
+                    doneCommand(line, storeTasks, filePath, tasksText);
+                } catch (DukeException e) {
+                    System.out.println(e.getMessage());
+                }
+                continue;
+            }
+            if (line.contains("deadline")) {
+                try {
+                    deadlineCommand(line, storeTasks, filePath, tasksText);
+                } catch (DukeException e) {
+                    System.out.println(e.getMessage());
+                }
+
+                continue;
+            }
+            if (line.contains("todo")) {
+                try {
+                    todoCommand(line, storeTasks, filePath, tasksText);
+
+                } catch (DukeException e) {
+                    System.out.println(e.getMessage());
+                }
+                continue;
+            }
+            if (line.contains("event")) {
+                try {
+                    eventCommand(line, storeTasks, filePath, tasksText);
+                } catch (DukeException e) {
+                    System.out.println(e.getMessage());
+                }
+                continue;
+            }
+            if (line.contains("delete")) {
+                try {
+                    deleteCommand(line, storeTasks, filePath, tasksText);
+                } catch (DukeException e) {
+                    System.out.println(e.getMessage());
+                }
+                continue;
+            }
+
+            try {
+                wrongCommand();
+            } catch (DukeException e) {
+                System.out.println(e.getMessage());
+            }
+        }
+
+        showEndScreen();
+    }
+
+    private static void showWelcomeScreen() {
+        String greet = " ____________________________________________________________\n"
```

Comment:
Can be better named as "greeting". Also, it can be defined as 1 huge constant as well.